Description: Fix load of RDP plugin when FreeRDP is compiled
 with -DSTATIC_CHANNELS=off
Author: Giovanni Panozzo

Index: remmina/remmina/src/remmina_plugin_manager.c
===================================================================
--- remmina.orig/remmina/src/remmina_plugin_manager.c
+++ remmina/remmina/src/remmina_plugin_manager.c
@@ -149,7 +149,7 @@ static void remmina_plugin_manager_load_
 	GModule *module;
 	RemminaPluginEntryFunc entry;
 
-	module = g_module_open(name, G_MODULE_BIND_LAZY | G_MODULE_BIND_LOCAL);
+	module = g_module_open(name, G_MODULE_BIND_LAZY);
 
 	if (!module)
 	{
